astropy/timeseries/tests/test_downsample.py

```diff
@@ -39,6 +39,26 @@ def test_reduceat():
     )
 
 
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
I wonder if it is cleaner to use https://numpy.org/doc/stable/reference/generated/numpy.testing.suppress_warnings.html instead and localize it to only the parts that you expect this warning and where it is actually okay to ignore.

Thanks!